components/ruby/mdl/Makefile

```diff
@@ -15,6 +15,7 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		mdl
 COMPONENT_VERSION=	0.4.0
+COMPONENT_REVISION=	1
 COMPONENT_SUMMARY=	Markdown lint tool
```

Comment:
I think you'll need to put this in a separate PR.

Reply:
Why?

Reply:
Because tt requires the newly introduced packages:

depend type=require fmri=library/ruby/kramdown
depend type=require fmri=library/ruby/kramdown-parser-gfm
depend type=require fmri=library/ruby/mixlib-cli
depend type=require fmri=library/ruby/mixlib-config

Reply:
No, as it's runtime dependency, not build-time.